index.html

```diff
@@ -1,20 +1,30 @@
 <html>
 <head>
-	<title>COP 4331 LAMP Stack Demo</title>
+	<title>Contact Manager - Group 21</title>
 	<script type="text/javascript" src="js/md5.js"></script>
 	<script type="text/javascript" src="js/code.js"></script>
-    <link href="css/styles.css" rel="stylesheet">	
+    <link href="css/styles.css" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
 </head>
 <body>
 
-<h1 id="title">COP 4331 LAMP Stack Demo</h1>
+<h1 id="title">Another Day in Paradise</h1>
 
 <div id="loginDiv">
-	<span id="inner-title">PLEASE LOG IN</span>
+	<span id="inner-title">PLEASE ENTER LOGIN CREDENTIALS</span>
 	<input type="text" id="loginName" placeholder="Username" /><br />
 	<input type="password" id="loginPassword" placeholder="Password"/><br />
-	<button type="button" id="loginButton" class="buttons" onclick="doLogin();"> Do It </button>
+	<button type="button" id="loginButton" class="buttons" onclick="doLogin();">Log In </button>
```

Comment:
You might want to use `button.addEventListener()` instead of `onclick=`; it is generally the preferred method in case you want to add multiple event handlers in the future. Not a blocker for merge but something to consider.